src/compiler/visitor.ts

```diff
@@ -549,6 +549,25 @@ namespace ts {
         return <Statement>singleOrUndefined(nodes) || createBlock(<NodeArray<Statement>>nodes);
     }
 
+    export function visitIterationBody(body: Statement, visitor: Visitor, context: TransformationContext): Statement {

```

Comment:
This needs to move to _visitorPublic.ts_ and should probably be made public just like `visitFunctionBody`, `visitParameterList`, etc.